ega-data-api-htsget/src/test/java/eu/elixir/ega/ebi/htsget/rest/V2IntegrationTest.java

```diff
@@ -0,0 +1,41 @@
+package eu.elixir.ega.ebi.htsget.rest;
+
+import eu.elixir.ega.ebi.htsget.HtsgetServiceApplication;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureWebMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@AutoConfigureWebMvc
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT, classes = {HtsgetServiceApplication.class})
+@TestPropertySource(locations = "classpath:application-test.properties")
+@Import(V2IntegrationTestConfig.class)
+public class V2IntegrationTest {
+
+    @Autowired
+    public TicketControllerV2 controller;
+
+    @Autowired
+    TestRestTemplate restTemplate;
+
+    @Test
+    @Ignore
+    public void runService() {
+
+        String versionStr = restTemplate.getForObject("/htsget/version", String.class);
+        Assert.assertEquals("v1.0.0", versionStr);
+
+        while (true) {
```

Comment:
This class/test looks like a bit unfinished, doesn't it?

Reply:
Yes, I made this test only as an easy way to run the service with mocks, so that I could test it with pyega3 from the command line. After I am back from holiday we will talk about how to test this service properly, and I think probably we will delete this test then.